mibidata/mibi_image.py

```diff
@@ -19,7 +19,12 @@
                                  'time_resolution', 'miscalibrated',
                                  'check_reg', 'filename', 'description',
                                  'version')
-
+APERTURE_MAP = {
```

Comment:
I'm fine to have this defined in one location, and this might well be the best place for it as we already use mibitracker-client in the mibitracker backend. But we should use one definition **everywhere** and mibi_shared defines it as:
APERTURE_1MM = u'A'
APERTURE_300UM = u'B'
APERTURE_100UM = u'C'
APERTURE_30UM = u'D'
APERTURES = (
    (APERTURE_1MM, u'1 mm'),
    (APERTURE_300UM, u'300 \u03BCm'),
    (APERTURE_100UM, u'100 \u03BCm'),
    (APERTURE_30UM, u'30 \u03BCm'),
)
Note the unicode and '1 mm' not '1mm'

Reply:
Done.